recipes/getdaft/meta.yaml

```
@@ -0,0 +1,66 @@
+{% set name = "getdaft" %}
+{% set version = "0.0.18" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/getdaft-{{ version }}.tar.gz
+  sha256: 1225c49f99c085f171eb0f0ce46d9a8eadf95b14b37de7181c4e54def62a897e
+
+build:
+  skip: true  # [not linux]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - arrow-cpp
+    - keyutils
+    - libffi
+    - libuuid
+  host:
+    - cython >=0.29.32
+    - numpy >=1.16.6
+    - pip
+    - poetry-core >=1.0.0
+    - poetry-dynamic-versioning
+    - python
+    - pyarrow ==6.0
```

Use this notation.
```suggestion
    - arrow-cpp
    - pyarrow 6
```

On the feedstock, it may be necessary to use the following skip as otherwise, the package would be build against all newer arrow releases. 

```
skip: true  # [arrow_cpp != "6.0.1"]
```

You might also try to build against all newer ones. As conda is a more potent package manager, we can support packages built for several arrow versions. We do this for example in the https://github.com/conda-forge/snowflake-connector-python-feedstock